responses/sk/HassLightSet.yaml

```diff
@@ -2,7 +2,7 @@ language: sk
 responses:
   intents:
     HassLightSet:
-      brightness: '{{ slots.name }} brightness set to {{ slots.brightness }}'
-      brightness_area: Brightness in {{ slots.area }} set to {{ slots.brightness }}
-      color: '{{ slots.name }} color set to {{ slots.color }}'
-      color_area: Color in {{ slots.area }} set to {{ slots.color }}
+      brightness: "Jas {{ slots.area }} nastavený na {{ slots.brightness }}"
+      brightness_area: "Jas {{ slots.area }} nastavený na {{ slots.brightness }}"
+      color: "Farba {{ slots.area }} zmenená na {{ slots.color }}"
+      color_area: "Farbe {{ slots.area }} zmenená na {{ slots.color }}"
```

Comment:
`slots.area` se používá k označení místnosti - to znamená u vět, kde ovládáme větší množství příslušenství (všechna světl v kuchyni) - pak jako slot dáme area.

`slot.name` je pro odpověď k ovládání jednoho zařízení - například garáž, nebo určité světlo.

Tedy `brightness` jakožto ovládání jednoho zařízení má `{{ slots.name }}`, kdežto `brightenss_area` - zde ovládáme jas v určité místnosti u včech zařízení má mít `{{ slots.area }}`

Reply:
fixnute